src/authentication/steps/Welcome.jsx

```diff
@@ -11,8 +12,54 @@ const getPlatformId = () =>
   isCordova() ? window.cordova.platformId : undefined
 
 export class Welcome extends Component {
+  registerRender = () => {
+    const { t, register, allowRegistration, hasSafariPlugin } = this.props
+
+    if (allowRegistration) {
+      return (
+        <a className={styles['link']} onClick={register}>
+          {t('mobile.onboarding.welcome.sign_up')}
+        </a>
+      )
+    }
+
+    const url = `https://manager.cozycloud.cc/cozy/create?pk_campaign=drive-${getPlatformId() ||
+      'browser'}`
+
+    if (hasSafariPlugin) {
+      const openManager = () => {
+        SafariViewController.show(
```

Comment:
It is a bit disturbing that with the hasSafariPlugin, it's easier for us to know that SafariViewController exists and is usable but then we still have to know about it.

I wonder if the cozy-device-helper should not directly expose a way of calling this component.